collections/business/general-notifications/event-general-notifications/account-statement-ready/account-statement-ready/in-app-notification/en_US/engagement-template.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "../../../../../../../../schema/v1/in-app-content.json",
```

Comment:
This does not look nice, right? Wondering if there is a way to improve it: `../../../../../../../../`